nextflow_schema.json

```diff
@@ -613,69 +572,53 @@
-                },
-                "validationFailUnrecognisedParams": {
-                    "type": "boolean",
-                    "fa_icon": "far fa-check-circle",
-                    "description": "Validation of parameters fails when an unrecognised parameter is found.",
-                    "hidden": true,
-                    "help_text": "By default, when an unrecognised parameter is found, it returns a warinig."
-                },
-                "validationLenientMode": {
-                    "type": "boolean",
+                "pipelines_testdata_base_path": {
+                    "type": "string",
                     "fa_icon": "far fa-check-circle",
-                    "description": "Validation of parameters in lenient more.",
-                    "hidden": true,
-                    "help_text": "Allows string values that are parseable as numbers or booleans. For further information see [JSONSchema docs](https://github.com/everit-org/json-schema#lenient-mode)."
+                    "description": "Base URL or local path to location of pipeline test dataset files",
+                    "default": "https://raw.githubusercontent.com/nf-core/test-datasets/methylseq",
+                    "hidden": true
                 }
             }
         }
     },
     "allOf": [
         {
-            "$ref": "#/definitions/input_output_options"
-        },
-        {
-            "$ref": "#/definitions/save_intermediate_files"
+            "$ref": "#/$defs/input_output_options"
         },
         {
-            "$ref": "#/definitions/reference_genome_options"
+            "$ref": "#/$defs/save_intermediate_files"
         },
         {
-            "$ref": "#/definitions/alignment_options"
+            "$ref": "#/$defs/reference_genome_options"
         },
         {
-            "$ref": "#/definitions/special_library_types"
+            "$ref": "#/$defs/alignment_options"
         },
         {
-            "$ref": "#/definitions/adapter_trimming"
+            "$ref": "#/$defs/special_library_types"
         },
         {
-            "$ref": "#/definitions/bismark_options"
+            "$ref": "#/$defs/adapter_trimming"
         },
         {
-            "$ref": "#/definitions/bwa_meth_options"
+            "$ref": "#/$defs/bismark_options"
         },
         {
-            "$ref": "#/definitions/qualimap_options"
+            "$ref": "#/$defs/bwa_meth_options"
         },
         {
-            "$ref": "#/definitions/skip_pipeline_steps"
+            "$ref": "#/$defs/qualimap_options"
         },
         {
-            "$ref": "#/definitions/generic_options"
+            "$ref": "#/$defs/skip_pipeline_steps"
         },
         {
-            "$ref": "#/definitions/max_job_request_options"
+            "$ref": "#/$defs/institutional_config_options"
         },
         {
-            "$ref": "#/definitions/institutional_config_options"
+            "$ref": "#/$defs/generic_options"
```

Comment:
Why did you exchange `institutional_config_options` and `generic_options`?

Reply:
🫣 happened during the template update